openmls/src/framing/codec.rs

```diff
@@ -210,3 +210,66 @@ impl tls_codec::Size for MlsMessage {
         }
     }
 }
+
+impl Size for MlsCiphertextContent {
+    fn tls_serialized_len(&self) -> usize {
+        self.content.tls_serialized_len() +
+           self.signature.tls_serialized_len() +
+            self.confirmation_tag.tls_serialized_len() +
+            // Note: The padding is appended as a "raw" all-zero byte slice
+            // with length `length_of_padding`. Thus, we only need to add
+            // this length here.
+            self.length_of_padding as usize
+    }
+}
+
+impl Serialize for MlsCiphertextContent {
+    fn tls_serialize<W: Write>(&self, writer: &mut W) -> Result<usize, Error> {
+        let mut written = 0;
+
+        written += self.content.tls_serialize(writer)?;
+        written += self.signature.tls_serialize(writer)?;
+        written += self.confirmation_tag.tls_serialize(writer)?;
+        let padding = vec![0u8; self.length_of_padding as usize];
```

Comment:
With all the castion I wonder why you picked `u16` for `length_of_padding`? Why don't you just use `usize`?

Reply:
I picked `u16` because it made the most sense to me for a padding. We do have `opaque ciphertext<V>;` and an implicit padding length now, though. Thus, in praxis, the maximum size of the padding is always 2^30 - the length of the other fields in `MLSCiphertextContent`. Using `u32` would be closer to the spec. `usize` is unnecessary large because it can not exceed 2^30 anyway. Thoughts?

Reply:
There's not really a reason not to use `usize`. It's not needed but you also don't safe anything by using something else.